meteor/server/api/blueprints/context/context.ts

```diff
@@ -184,30 +146,60 @@ export class StudioConfigContext implements IStudioConfigContext {
 	getStudioConfigRef(configKey: string): string {
 		return ConfigRef.getStudioConfigRef(this.studio._id, configKey)
 	}
-}
 
-export class StudioContext extends StudioConfigContext implements IStudioContext {
 	getStudioMappings(): Readonly<BlueprintMappings> {
 		return this.studio.mappings
 	}
 }
 
-/** Show Style Variant */
+export class StudioUserContext extends StudioContext implements IStudioUserContext {
+	public readonly notes: INoteBase[] = []
+	private readonly tempSendNotesIntoBlackHole: boolean
 
-export class ShowStyleContext extends StudioContext implements IShowStyleContext {
-	readonly notesContext: NotesContext
+	constructor(contextInfo: UserContextInfo, studio: Studio) {
+		super(contextInfo, studio)
+		this.tempSendNotesIntoBlackHole = contextInfo.tempSendUserNotesIntoBlackHole ?? false
+	}
+
+	notifyUserError(message: string, params?: { [key: string]: any }): void {
+		if (this.tempSendNotesIntoBlackHole) {
```

Comment:
Should this temporary feature be removed?

Reply:
we dont have a way of getting these to the user currently, but they really should be.
so this is a hack to make them go somewhere (rather than into the void) until a follow up task is written and done

Reply:
but I am thinking I will end up making the blueprints do a log* everytime they do a notifyUser* so that we have a record of the note with more technical details, so maybe this isnt necessary?